app/src/main/java/com/infomaniak/drive/ui/home/RootFilesFragment.kt

```diff
@@ -82,46 +89,96 @@ class RootFilesFragment : Fragment() {
         )
 
         setupRootPendingFilesIndicator(mainViewModel.pendingUploadsCount, rootFilesUploadFileInProgressView)
+
+        if (fileListViewModel.justLaunched.not()) {
```

Comment:
Sounds like a habit thing rather than a clarity thing to me: `.not()` is more visible since it's a bit longer, and it's just next to the boolean expression it inverts, meaning you don't need to remember about the little `!` as one keeps reading the line.

Reply:
It is a habit that define clarity ^^
I personnally find it way more clear with the `!`, probably because of the habit 
We'll discuss it also in meeting